infra/aws/govcloud.md

```diff
@@ -0,0 +1,199 @@
+    "State": "SUCCEEDED",
+    "RequestedTimestamp": 1582040671.446,
+    "CompletedTimestamp": 1582040736.794,
+    "AccountId": "123456789000",
+    "GovCloudAccountId": "000987654321"
+  }
+}
+```
+
+In that output, `AccountId` is the account number for the dummy account,
+and the `GovCloudAccountId` is the account number for your new GovCloud
+account.
+
+Now that you have that, you can assume the `OrganizationAccountAccessRole`
+in the new account from an account with sufficient privileges in the
+*GovCloud* organization master account. *Note that this is the case even
+though at this point the GovCloud organization does not show up if you
+look at the organization membership in the console.*
+
+### Adding Your New GovCloud Account to Your GovCloud Organization
+
+To add the account you just created to your GovCloud organization, run
+this command with your appropriate credentials from the GovCloud
+organization's master account.
+
+```console
+$ aws organizations invite-account-to-organization \
+  --target Id=000987654321,Type=ACCOUNT
+```
+
+This will return output that looks like this:
+
+```text
+{
+  "Handshake": {
+    "Id": "h-d6ac6154ef6a42e68513d0d8460af49a",
+
+...
+```
+
+Using the credentials in the new account, run this command:
+
+```console
+$ aws organizations accept-handshake \
+  --handshake-id h-d6ac6154ef6a42e68513d0d8460af49a
+```
+
+The account will now be in your GovCloud organization. You will want to
+import this account into your Terraform state for the GovCloud
+organization master account like so:
+
+```hcl
+resource "aws_organizations_account" "spacecats_govcloud_id" {
+  name      = "spacecats-govcloud-id"
+  email     = "spacecats-infra+spacecats-govcloud-id@example.com"
+  parent_id = aws_organizations_organizational_unit.spacecats_govcloud.id
+  tags = {
+    Automation = "Terraform"
+  }
+}
+```
+
+Then run this command:
+
+```console
+$ terraform import aws_organizations_account.spacecats_govcloud_id 000987654321
+```
+
+### Handling GovCloud Dummy Accounts in AWS Commercial
+
+As stated above, when you use the AWS CLI to create your new GovCloud
+accounts, it will also create a dummy account with the same account name
+in your commercial AWS organization. In order to prevent misuse of these
+accounts (and any confusion, since they have the same account alias as
+your GovCloud account), we can lock these down with the `suspended` OU
+we usually create in the commercial AWS organization.
+
+To do this, import the account's Terraform resource into your commercial
+AWS organization by using the same sort of procedure described above to
+import the new GovCloud account into the GovCloud organization, but
+replace the `parent_id` parameter with the Terraform resource describing
+your `suspended` OU -- usually something like
+`aws_organizations_organizational_unit.suspended.id`. This will make it
+impossible to use that account for anything else, and has no effect on
+the corresponding GovCloud account.
```

Comment:
Can you also link the terraform docs for import of this resource?

Reply:
What do you mean? Do you just mean this page: https://www.terraform.io/docs/import/usage.html or something else? I gave an example of how to do this in the previous section, where I showed how to do it for the GovCloud account.

Reply:
Not the `terraform import` docs but the page for the terraform resource. Like this one: https://www.terraform.io/docs/providers/aws/r/organizations_organizational_unit.html#import